kcp2k/Assets/kcp2k/highlevel/KcpConnection.cs

```diff
@@ -241,6 +241,8 @@ bool ReceiveNextReliable(out KcpHeader header, out ArraySegment<byte> message)
                 }
             }
 
+            // Create empty message result
+            message = new ArraySegment<byte>(kcpMessageBuffer, 0, 0);
```

Comment:
Wouldn't `Array.Empty<byte>()` be better in this case?

Reply:
Good point, that for sure helps readability!